M/MonteCarloMeasurements/Compat.toml

```diff
@@ -3,3 +3,6 @@ julia = "1.1.0-1"
 
 ["0.1.2"]
 julia = "1.0.0-1"
+
+["0.1.3"]
+julia = ["1.0.0-1"]
```

Comment:
What happened here @StefanKarpinski ? Whats the difference between this and the range on L5?

Reply:
It’s the same. @nkottary seems to have deployed the fix that @pfitzseb wrote but not the complete one that I provided.